### ISSUE TITLE
EMC Dupe

### ISSUE THREAD

**Issue by TeamDman** [sinkillerj/ProjectE#185]
Post alchemical tome burnage, (sidebug, crafting tome did not require omega klein stars to be full, IIRC EE2 required them to be full) when in my portable transmutation tablet I noticed that it supplied stacks size that crafting returned, I did not notice this behavior before. Oddly, this did not happen with vanilla items. For example, when I search torch I see vanilla torches (stack size 1), TiCon torches (stack size 4) and Interdiction Torches (stack size 2). Using interdiction torches as a example, when I pulled out the stack with shift clicking, it would give a full stack of the item and when I burn the stack my stored EMC returned to normal. However, when I picked them up manually, it would give me 2 at a time for the cost of one, allowing them to be burned effectively duplicating interdiction torches' EMC value.

TL;DR Transmutation tablet shows items w/ stacksize >= 1 which can be picked up for the cost of 1 then burned.
